OWNERS_ALIASES

```diff
@@ -14,50 +14,57 @@ aliases:
     - ixdy
     - mikedanese
     - tpepper
+  release-team:
+    - aishsundar
+    - jberkus
+    - justaugustus
```

Comment:
I thought we had discussed making this consist solely of emeritus release leads?

Reply:
@spiffxp -- I'm not sure that we said _solely_; just that we'd initially seed from the n-4 RT Leads. I'd also like to participate as a subproject owner.

Reply:
can we comment there saying n-4 RT leads. It makes it easier to maintain. the `release-team` name for an emeritus group alias is confusing.